staging/src/k8s.io/component-base/metrics/metric.go

```diff
@@ -159,6 +161,13 @@ func (r *lazyMetric) Create(version *semver.Version) bool {
 			r.self.initializeMetric()
 		}
 	})
+	sl := r.stabilityLevel
+	deprecatedV := r.self.DeprecatedVersion()
+	dv := ""
+	if deprecatedV != nil {
+		dv = deprecatedV.String()
+	}
+	defer registeredMetrics.WithLabelValues(string(sl), dv).Inc()
```

Comment:
just a question. Why use `defer` here?

Reply:
ah that was from a move, i don't think it matters in this exact spot in practice.